repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -83,6 +83,7 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
       case Success(serviceClientResponse) =>
         serviceClientResponse.getStatus match {
           case statusCode if statusCode >= 200 && statusCode < 300 =>
+            // TODO: Handle character encoding set in the content-type header rather than relying on the default system encoding
```

Comment:
So yeah... we definitely are not explicit about character encoding when parsing the response from Identity. We just assume the JVM default encoding. We also don't send the `accept-charset` header in our request, but could do so if we wanted to express a preference for a certain encoding.